spec/arrays.dd

```diff
@@ -576,8 +576,8 @@ Returns an array literal with each element of the literal being the $(D .init) p
         the number of bytes per array element.)
         $(TROW $(D .length), Returns the number of elements in the array.
         This is a fixed quantity for static arrays. It is of type $(D size_t).)
-        $(TROW $(D .capacity), Returns the number of elements that can be appended to the array without reallocating.
-        $(D .capacity) is always $(D 0) for static arrays because their size cannot be modified.)
+        $(TROW $(D .capacity), Always $(D 0) for static arrays because their size cannot be modified.
```

Comment:
well....
```d
struct S
{
    int[4] x;
}
void main()
{
    auto s = [S()];
    import std.stdio;
    writeln(s[0].x.capacity); // 7
}
```

I think the capacity should be removed as a property for static arrays. It's not a property of static arrays, but of dynamic arrays. The reason it works is because D implicitly casts the static array to a dynamic array.

Reply:
Interesting, thanks. I have removed the static array property.